src/iOS.Autofill/LoginListViewController.cs

```diff
@@ -350,30 +361,31 @@ private async Task SavePasskeyAsNewLoginAsync()
             }
         }
 
-        private async Task<bool> VerifyUserAsync()
+        private async Task<CancellableResult<bool>> VerifyUserAsync()
         {
             try
             {
                 if (Context?.PasskeyCreationParams is null)
                 {
-                    return false;
+                    return new CancellableResult<bool>(false);
                 }
 
-                return await _userVerificationMediatorService.Value.VerifyUserForFido2Async(
-                    new Fido2UserVerificationOptions(
-                        false,
-                        Context.PasskeyCreationParams.Value.UserVerificationPreference,
-                        Context.VaultUnlockedDuringThisSession,
-                        Context.PasskeyCredentialIdentity?.RelyingPartyIdentifier)
-                    );
+                return await _userVerificationMediatorService.Value.VerifyUserForFido2Async(Fido2UserVerificationOptions);
             }
             catch (Exception ex)
             {
                 LoggerHelper.LogEvenIfCantBeResolved(ex);
-                return false;
+                return new CancellableResult<bool>(false);
             }
         }
 
+        private Fido2UserVerificationOptions Fido2UserVerificationOptions => new Fido2UserVerificationOptions(
+            false,
+            Context.PasskeyCreationParams.Value.UserVerificationPreference,
```

Comment:
❓ There are other places in this class where `Context` can be `null` isn't the case here?

Reply:
Here if it's null it should just break the flow because we wouldn't have the necessary parameters to build the options and perform the user verification. It's unlikely to happen and also on the VerifyUserAsync we're checking the nullability before calling this. The only part that will be a concern is when checking the UV enforcement but I think it should just throw and it will display a message to the user.
Nonetheless, I can add an argument null check of the Context so if we get this in AppCenter we know that's the specific reason, thanks for catching this!